src/compiler/JSCodeGenVisitor.js

```diff
@@ -265,6 +276,10 @@ module.exports = class JSCodeGenVisitor {
       this._dom.doctype(cmd);
     } else if (cmd instanceof Comment) {
       this._dom.comment(cmd);
+    } else if (cmd instanceof ExternalCode) {
+      this.pushBlock('foo');
```

Comment:
What does this do? 🤔

Reply:
it opens a new template block with the name 'foo'. maybe the name can be left empty.